contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -70,6 +70,7 @@
 #include "catalog/pg_trigger.h"
 #include "catalog/pg_constraint.h"
 #include "parser/parse_oper.h"
+#include "../src/dbcmds.h"
```

Comment:
the header should be in lexicographical order

Reply:
Done